File: src/main/java/io/earlisreal/ejournal/parser/Broker.java
package io.earlisreal.ejournal.parser;

public enum Broker {

    AAA("AAA", System.lineSeparator() + "AAA" + System.lineSeparator()),
    YAPSTER("2TradeAsia", "YAPSTER e-TRADE");

    private final String name;
    private final String uniqueIdentifier;

    Broker(String name, String uniqueIdentifier) {
        this.name = name;
        this.uniqueIdentifier = uniqueIdentifier;
    }

    public String getName() {
        return name;
    }

    public String getUniqueIdentifier() {
        return uniqueIdentifier;
    }

}

File: shared/src/commonMain/kotlin/io/earlisreal/ejournal/domain/model/Broker.kt
package io.earlisreal.ejournal.domain.model

enum class Broker(
    val id: String,
    val label: String,
) {
    ALPACA("alpaca", "Alpaca"),
    MOOMOO("moomoo", "Moomoo"),
    TRADEZERO("tradezero", "TradeZero"),
}
